### PR TITLE
[INLONG-1086][Doc] Add documents for DolphinScheduler offline scheduler

### LINKED ISSUE #1086: [Feature][Doc] Add documents for DolphinScheduler offline scheduler
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by emptyOVO**
Fixes #1086 

### Motivation
* Add doc for DolphinScheduler offline schedule
* Modify the original `Pulsar-MySQL` example to built-in engine example

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
    - *Added integration tests for end-to-end deployment with large payloads (10MB)*
    - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/quick_start/offline_data_sync/img/dolphinscheduler/ds_schedule_success.png
Comment (Zkplo) [inline docs/quick_start/offline_data_sync/img/dolphinscheduler/ds_schedule_success.png:1]: `BoundaryType` does not match the latest code.

**Comment by Zkplo** [comment]
The explanatory document should be added in both Chinese and English.

File: docs/quick_start/offline_data_sync/img/dolphinscheduler/ds_schedule_success.png
Comment (emptyOVO) [reply docs/quick_start/offline_data_sync/img/dolphinscheduler/ds_schedule_success.png:1]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: 使用第三方 DolphinScheduler 调度引擎
+sidebar_position: 1
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md:3]:
```suggestion
sidebar_position: 2
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+在接下来的内容中，我们将介绍如何在 Apache InLong 中集成第三方调度引擎 DolphinScheduler 来创建离线数据同步。
+
+## ## 环境部署
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md:8]:
```suggestion
## 环境部署
```

File: docs/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+title: Use DolphinScheduler third-party scheduling engine
+sidebar_position: 1
```
Comment (aloyszhang) [inline docs/quick_start/offline_data_sync/dolphinscheduler_example.md:3]:
```suggestion
sidebar_position: 2
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- [Bare Metal 部署](deployment/bare_metal.md)
+
+### 添加 Connectors
+
+下载与 Flink 版本对应的 [connectors](https://inlong.apache.org/zh-CN/downloads)，解压后将 `sort-connector-jdbc-[version]-SNAPSHOT.jar` 放在 `/inlong-sort/connectors/` 目录下。
+> 当前 Apache InLong 的离线数据同步能力只支持 Flink-1.18 版本，所以请下载 1.18 版本的 connectors。
+
+### 在 DolphinScheduler 上的操作
+
+在使用 DolphinScheduler 作为调度引擎之前，请确保手头有正常可用的 DolphinScheduler。如果您需要为自己部署一个 DolphinScheduler，请参考 [DolphinScheduler 官方文档](https://dolphinscheduler.apache.org/zh-cn)。
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md:24]:
```suggestion
在使用 DolphinScheduler 作为调度引擎之前，请确保有可以提供服务的 DolphinScheduler 。如果您需要为自己部署一个 DolphinScheduler，请参考 [DolphinScheduler 官方文档](https://dolphinscheduler.apache.org/zh-cn)。
```

File: docs/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+In the following content, we will introduce how to use DolphinScheduler third-party schedule engine in Apache InLong to create offline data synchronization.
```
Comment (aloyszhang) [inline docs/quick_start/offline_data_sync/dolphinscheduler_example.md:6]:
```suggestion
In the following content, we will introduce how to use DolphinScheduler, a third-party schedule engine, in Apache InLong to create offline data synchronization.
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md (hunk 1 above)
Comment (emptyOVO) [reply i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md]: thanks, done

**Review by aloyszhang — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md [hunk 6]
```diff
@@ -26,44 +26,48 @@ Password: inlong
...
 常规调度配置需要设置以下参数：
 - 调度单位：支持分钟、小时、天、月、年以及单次，单次表示只执行一次
 - 调度周期：表示两次任务调度之间的时间间隔
 - 延迟时间：表示任务启动的延迟时间
 - 有效时间：包括起始时间和结束时间，调度任务只会在这个时间范围内执行
 
-![Create Offline Group](img/pulsar_mysql/normal_schedule.png)
+![Create Offline Group](img/pulsar_mysql/quartz/normal_schedule.png)
 
 crontab调度需要设置以下参数：
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md:60]:
```suggestion
crontab 调度需要设置以下参数：
```

Comment (emptyOVO) [reply i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md [hunk 7]
```diff
@@ -80,25 +88,25 @@ CREATE TABLE sink_table (
 
 数据目标中 点击 【新建】→【MySQL】，配置数据目标名称、库名和表名（test.sink_table）等信息。
 
-![Create Sink](img/pulsar_mysql/sink.png)
+![Create Sink](img/pulsar_mysql/quartz/sink.png)
 
 ### 配置字段信息
 
 分别在 【源字段】 和 【目标字段】中配置 Schema 映射信息，完成后点击 【提交审批】
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md:95]:
```suggestion
分别在 【源字段】 和 【目标字段】中配置 Schema 映射信息，完成后点击 【提交审批】。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md [hunk 8]
```diff
@@ -80,25 +88,25 @@ CREATE TABLE sink_table (
...
 
-![Create Source Fields](img/pulsar_mysql/source_field.png)
+![Create Source Fields](img/pulsar_mysql/quartz/source_field.png)
 
-![Create Sink Fields](img/pulsar_mysql/sink_field.png)
+![Create Sink Fields](img/pulsar_mysql/quartz/sink_field.png)
 
 ### 审批数据流
 
 页面点击【审批管理】->【我的审批】->【详情】->【通过】
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md:103]:
```suggestion
页面点击【审批管理】->【我的审批】->【详情】->【通过】。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md [hunk 9]
```diff
@@ -26,44 +26,52 @@ Password: inlong
...
-![Create Offline Group](img/pulsar_mysql/create_offline_group.png)
+![Create Offline Group](img/pulsar_mysql/quartz/create_offline_group.png)
 
 ### 配置调度规则
-在同步类型勾选为“离线”之后，就可以配置离线任务的调度规则，目前支持两种：常规和 crontab。
+在同步类型勾选为“离线”之后，就可以配置离线任务的调度规则，调度规则主要由两个部分组成，分别为【调度引擎】和【调度类型】。
 
-常规调度配置需要设置以下参数：
+#### 调度引擎
+Apache InLong 提供了多种调度引擎供用户选择，Quartz 是 Apache InLong 内置的调度引擎，这里使用 Quartz 来处理任务
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/quartz_example.md:49]:
```suggestion
Apache InLong 提供了多种调度引擎供用户选择，Quartz 是 Apache InLong 内置的调度引擎，这里使用 Quartz 来处理任务。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+审批数据流后，返回【数据同步】页面，等待任务配置成功，配置成功后，DolphinScheduler 将定期回调 InLong Manager，并由 InLong Manager 周期提交 Flink Batch Job 到 Flink 集群。
+
+![DolphinScheduler Schedule Process](img/pulsar_mysql/dolphinscheduler/ds_schedule_process.png)
+
+![DolphinScheduler Process Success](img/pulsar_mysql/dolphinscheduler/ds_process_success.png)
+
+![DolphinScheduler Process Instance](img/pulsar_mysql/dolphinscheduler/ds_process_instance.png)
+
+查看 DolphinScheduler 任务实例日志。输出如下日志表示配置成功
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md:68]:
```suggestion
查看 DolphinScheduler 任务实例日志，输出如下日志表示配置成功。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+* `schedule.engine.dolphinscheduler.url` : DolphinScheduler 部署的 url，一般格式为 http://{ip}:{port}/dolphinScheduler
+* `schedule.engine.dolphinscheduler.token` : 上文中在 DolphinScheduler 令牌管理器页面中生成的 Token。
+
+![InLong Manager Configuration](img/pulsar_mysql/dolphinscheduler/inlong_manager_conf.png)
+
+完成这些操作后，重新启动 InLong Manager 以确保配置成功启用。
+
+### 在离线同步任务中使用 DolphinScheduler
+
+在配置离线同步任务时，在选择调度引擎时选择 DolphinScheduler，然后配置其他参数
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md:54]:
```suggestion
在配置离线同步任务时，在选择调度引擎时选择 DolphinScheduler，然后配置其他参数。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+![DolphinScheduler Token Generate](img/pulsar_mysql/dolphinscheduler/ds_token_generate.png)
+
+按照图中步骤设置 token 参数，包括 [过期时间]、[用户]，并生成令牌。
+
+![DolphinScheduler Token Copy](img/pulsar_mysql/dolphinscheduler/ds_token_copy.png)
+
+### 修改 InLong Manager 中的配置
+
+对于第三方调度引擎，我们需要在 InLong Manager 中修改配置
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md:40]:
```suggestion
对于第三方调度引擎，我们需要在 InLong Manager 中修改配置。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+
+### 在 DolphinScheduler 上的操作
+
+在使用 DolphinScheduler 作为调度引擎之前，请确保有可以提供服务的 DolphinScheduler 。如果您需要为自己部署一个 DolphinScheduler，请参考 [DolphinScheduler 官方文档](https://dolphinscheduler.apache.org/zh-cn)。
+
+![DolphinScheduler Security](img/pulsar_mysql/dolphinscheduler/ds_security.png)
+
+![DolphinScheduler Token Manager](img/pulsar_mysql/dolphinscheduler/ds_token_manager.png)
+
+进入令牌管理器页面创建一个令牌供 InLong 进行访问
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0.0/quick_start/offline_data_sync/dolphinscheduler_example.md:30]:
```suggestion
进入令牌管理器页面创建一个令牌供 InLong 进行访问。
```

File: docs/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+### Add Connectors
+
+Download the [connectors](https://inlong.apache.org/downloads/) corresponding to Flink version, and after decompression, place `sort-connector-jdbc-[version]-SNAPSHOT.jar` in `/inlong-sort/connectors/` directory.
+
+> Currently, Apache InLong's offline data synchronization capability only supports Flink-1.18, so please download the 1.18 version of connectors.
+
+### Operations on DolphinScheduler
+
+Before using DolphinScheduler as your scheduling engine, please make sure you have a working DolphinScheduler on hand. If you need to deploy a DolphinScheduler for yourself, please refer to the [DolphinScheduler Official Document](https://dolphinscheduler.apache.org/zh-cn)
```
Comment (fuweng11) [inline docs/quick_start/offline_data_sync/dolphinscheduler_example.md:25]:
```suggestion
Before using DolphinScheduler as your scheduling engine, please make sure you have a working DolphinScheduler on hand. If you need to deploy a DolphinScheduler for yourself, please refer to the [DolphinScheduler Official Document](https://dolphinscheduler.apache.org/zh-cn).
```

File: docs/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+
+![DolphinScheduler Security](img/pulsar_mysql/dolphinscheduler/ds_security.png)
+
+![DolphinScheduler Token Manager](img/pulsar_mysql/dolphinscheduler/ds_token_manager.png)
+
+Go into Token Manager page to create a token for InLong to access
```
Comment (fuweng11) [inline docs/quick_start/offline_data_sync/dolphinscheduler_example.md:31]:
```suggestion
Go into Token Manager page to create a token for InLong to access.
```

File: docs/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+
+![DolphinScheduler Token Generate](img/pulsar_mysql/dolphinscheduler/ds_token_generate.png)
+
+Set parameters for the token according to the steps in the figure, include [Expiration Time], [User], then generate a token.
+
+![DolphinScheduler Token Copy](img/pulsar_mysql/dolphinscheduler/ds_token_copy.png)
+
+### Modify configuration in InLong Manager
+
+For third-party scheduling engine, we need to modify configurations in manager
```
Comment (fuweng11) [inline docs/quick_start/offline_data_sync/dolphinscheduler_example.md:41]:
```suggestion
For third-party scheduling engine, we need to modify configurations in manager.
```

File: docs/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+* `schedule.engine.dolphinscheduler.url` : DolphinScheduler deployment url, general format is http://{ip}:{port}/dolphinscheduler
+* `schedule.engine.dolphinscheduler.token` : Token you just generated in Token Manager of DolphinScheduler
+
+![InLong Manager Configuration](img/pulsar_mysql/dolphinscheduler/inlong_manager_conf.png)
+
+After doing this, restart the InLong Manager to ensure the configuration is enabled.
+
+### Use DolphinScheduler in offline synchronization
+
+During configure the offline synchronization task, choose DolphinScheduler when selecting the scheduling engine, then configure other parameters
```
Comment (fuweng11) [inline docs/quick_start/offline_data_sync/dolphinscheduler_example.md:55]:
```suggestion
During configure the offline synchronization task, choose DolphinScheduler when selecting the scheduling engine, then configure other parameters.
```

File: docs/quick_start/offline_data_sync/dolphinscheduler_example.md [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+
+After approval data flow, return to the [Synchronization] page and wait for the task configuration to succeed. Once configured successfully, the DolphinScheduler will periodically calls back InLong to synchronize offline data and the Manager will periodically submit Flink Batch Jobs to the Flink cluster.
+
+![DolphinScheduler Schedule Process](img/pulsar_mysql/dolphinscheduler/ds_schedule_process.png)
+
+![DolphinScheduler Process Success](img/pulsar_mysql/dolphinscheduler/ds_process_success.png)
+
+![DolphinScheduler Process Instance](img/pulsar_mysql/dolphinscheduler/ds_process_instance.png)
+
+View the DolphinScheduler task instance logs. The following logs indicate that the configuration is successful
```
Comment (fuweng11) [inline docs/quick_start/offline_data_sync/dolphinscheduler_example.md:69]:
```suggestion
View the DolphinScheduler task instance logs. The following logs indicate that the configuration is successful.
```

Comment (emptyOVO) [reply docs/quick_start/offline_data_sync/dolphinscheduler_example.md]: done

**Review by dockerzhang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Review by fuweng11 — APPROVED** [review]

**Pull request merged**